public/app/features/query/state/queryAnalytics.ts

```diff
@@ -32,8 +32,8 @@ export function emitDataRequestEvent(datasource: DataSourceApi) {
       duration: data.request.endTime! - data.request.startTime,
```

Comment:
I'd revert the changes in this file, i don't think we actually need query analytics for the correlations setup page, i think the important bit was to set `data.request.app` to  `CoreApp.Correlations` so that the code below is not triggered. wdyt @ifrost

Reply:
or better, something like 
```
if(explore){
  doExploreStuff();
} else if (dashboard) {
  doDashboardStuff();
}
```

Reply:
If we don't need the query analytics I'd go with Gio's second suggestion the first one is breaking the tests.

Reply:
This tracking is important for Usage Insights (https://github.com/grafana/grafana/pull/59931). I think we should keep it.

Reply:
After consulting with Piotr I reverted the changes from the commit below.
<img width="663" alt="Screenshot 2023-01-12 at 10 33 11" src="https://user-images.githubusercontent.com/48948963/212030713-7b6ac507-947e-4104-ad9f-3b506d7aef48.png">